js/build/shortcode-ui.js

```diff
@@ -1685,372 +1547,253 @@ var wp = (typeof window !== "undefined" ? window['wp'] : typeof global !== "unde
-	initialize : function() {
-		_.defaults(this.options, {
-			requires : false
-		});
-		// Call 'initialize' directly on the parent class.
-		wp.media.view.Toolbar.prototype.initialize.apply(this, arguments);
-	},
 
 	refresh : function() {
-		var action = this.controller.state().props.get('action');
-		if( this.get('insert') ) {
-			this.get('insert').model.set('disabled', action == 'select');
+
+		var action = this.controller.content.mode();
+
+		if ( action ) {
+			this.get('insert').model.set( 'disabled', action !== 'shortcode-ui-content-edit' );
 		}
+
 		/**
 		 * call 'refresh' directly on the parent class
 		 */
 		wp.media.view.Toolbar.prototype.refresh.apply(this, arguments);
+
 	}
 });
 
 module.exports = Toolbar;
 
 }).call(this,typeof global !== "undefined" ? global : typeof self !== "undefined" ? self : typeof window !== "undefined" ? window : {})
-},{}],20:[function(require,module,exports){
+},{}],19:[function(require,module,exports){
 (function (global){
-var wp = (typeof window !== "undefined" ? window['wp'] : typeof global !== "undefined" ? global['wp'] : null);
-sui = require('./../utils/sui.js');
+var wp         = (typeof window !== "undefined" ? window['wp'] : typeof global !== "undefined" ? global['wp'] : null),
+	$          = (typeof window !== "undefined" ? window['jQuery'] : typeof global !== "undefined" ? global['jQuery'] : null),
+	sui        = require('./../utils/sui.js'),
+	State      = require('./../controllers/media-controller.js'),
+	Toolbar    = require('./media-frame-toolbar.js'),
+	ListView   = require('./insert-shortcode-list.js'),
+	EditView   = require('./edit-shortcode-form.js'),
+	Frame      = wp.media.view.Frame;
 
-var SearchShortcode = wp.media.view.Search.extend({
-	tagName:   'input',
-	className: 'search',
-	id:        'media-search-input',
-	
-	initialize: function( options ) {
-		this.shortcodeList = options.shortcodeList;
-	}, 
+var ShortcodeUiFrame = Frame.extend( {
 
-	attributes: {
-		type:        'search',
-		placeholder: shortcodeUIData.strings.search_placeholder
-	},
+	className: 'media-frame',
+	regions:   [ 'title', 'content', 'toolbar' ],
+	template:  wp.template('media-frame'),
 
-	events: {
-		'keyup':  'search',
-	},
+	initialize: function() {
 
-	/**
-	 * @returns {wp.media.view.Search} Returns itself to allow chaining
-	 */
-	render: function() {
-		this.el.value = this.model.escape('search');
-		return this;
-	},
-	
-	refreshShortcodes: function( shortcodeData ) {
-		this.shortcodeList.refresh( shortcodeData );
-	},
+		Frame.prototype.initialize.apply( this, arguments );
 
-	search: function( event ) {
-		if ( '' === event.target.value ) {
-			this.refreshShortcodes( sui.shortcodes );
-		} else {
-			this.refreshShortcodes( this.controller.search( event.target.value ) );
+		_.bindAll( this, 'select', 'reset' );
+
+		this.options = _.defaults( this.options, {
+			state:          'shortcode-ui',
+			modal:          true,
+			title:          'Insert',
+			updateTitle:    'Update',
```

Comment:
Fixed. These should be empty strings as defaults, with localized labels passed in when creating the view. I've updated this to ensure they are.